gourmet/GourmetRecipeManager.py

```diff
@@ -739,15 +737,29 @@ def print_recs (self, *args):
             change_units=self.prefs.get('readableUnits',True)
             )
 
+    _import_manager = None
```

Comment:
can we have that double-underscored? It's then renamed by the interpreter to be not so easily accessible by outside callers.

Reply:
Sure, done. I tend to avoid double underscore because the name mangling is an extra confusion to think about, but that's a matter of taste.

There's also the [cached_property](https://pypi.org/project/cached-property/) package which can do this neatly, but I didn't want to introduce an extra dependency now to save a few lines.